lib/utils/qemu.coffee

```diff
@@ -67,20 +67,43 @@ platformNeedsQemu = ->
 	os = require('os')
 	os.platform() == 'linux'
 
-installQemu = ->
+installQemu = (arch) ->
 	request = require('request')
 	fs = require('fs')
 	zlib = require('zlib')
+	tar = require('tar')
```

Comment:
We have `tar-stream` as a dependency already, would it be possible to use this instead? From the usage below it looks like it would be.

Reply:
Yeah, I initially tried using `tar-stream`, as I didn't want to add unnecessary dependencies, but was having trouble getting it to work with the existing in-memory approach of extracting the file to `installStream` (can't remember what the exact issue was). I'll circle back to it and try again.